0006/README.md

```diff
@@ -0,0 +1,364 @@
+│  ├─ .ruby-version
+│  ├─ _config.yml
+│  ├─ Gemfile
+│  ├─ Gemfile.lock
+│  ├─ README.md
+│  └─ <other-unversioned-files>.md
+├─ .gitignore
+├─ LICENSE.md
+├─ README.md
+└─ ...
+
+# Files were formerly in main/docs/*/v1, such as:
+# `docs/spec/v1.0/*`,
+# `docs/provenance/v1/*`,
+# `docs/verification_summary/v1/*`
+slsa-framework/slsa:release/v1.0
+└─ <v1.0-versioned-files>.md
+
+# Files were formerly in main/docs/*/v0.1, such as:
+# `docs/spec/v0.1/*`,
+# `docs/provenance/v0.1/*`,
+# `docs/verification_summary/v0.1/*`
+slsa-framework/slsa:release/v0.1
+└─ <v0.1-versioned-files>.md
+```
+
+#### Main Branch
+
+The directory structure of `main` stays the same as the repo has right now.
+
+*What is changing*
+
+-   `/docs/spec`, `/docs/provenance`, `/docs/verification_summary`, and
+    `/docs/github-actions-workflow` are removed from `main`.
+-   `/docs/v1.0` and `/docs/v0.1` are two Git subtrees that have the same
+    content as [version branches](#version-branches). Using Git subtrees is to
+    ensure a consistent user experience for the community, so that a contributor
+    only needs to make changes in `main`.
+-   When detecting a push in the versioned directories, a Github Workflow syncs
+    the same change into the version branches. For example:
+
+    ```yaml
+    # sync_branch.yml lives in .github/workflows.
+    on:
+      push:
+        branches: [ main ]
+        paths: [ docs/v+/ ]
+    ```
+
+-   Every permalink is updated to use the dir structure described in
+    [the website](#the-website) section.
+
+#### Version Branches
+
+Every prior or future specification has a version branch.
+
+*What is changing*
+
+-   `release/<version|version-draft>` is the branch pattern for a prior or
+    future `version`, where the `version` must contain both major and minor.
+-   At the top level directory, a branch has all the versioned file from
+    `/docs/spec/<version>/*`, `/docs/provenance/<version>.md`,
+    `/docs/verification_summary/<version>.md`, and
+    `/docs/github-actions-workflow/<version>.md`. All specification files in the
+    same branch share the same version.
+-   When a new version branch is created, the repo owner is responsible to
+    manage Git subtrees and other website configs for the branch in `main`.
+
+#### Code Contribution
+
+The process should remain the same as it is for today. Github subtrees and
+version branches are agnostic to community contributors.
+
+-   If a new version is desired, a contributor should raise the request either
+    in the SLSA weekly meeting or create an issue in the repo. The repo owner
+    evaluates the request, and sets up the new version in the repo.
+-   If a change is desired in the prior or future version, the contributor
+    should fork the repo, clone it locally if they are familiar with git, make
+    the change in the `main:docs/<version>` directories, and send a PR against
+    `main`.
```

Comment:
I don't understand this. If I want to make a change to v0.1 for instance. Wouldn't I need to submit my change against the release/0.1 branch rather than main?